### ISSUE TITLE
gymdetails table doesn't get created, throws Exception

### ISSUE THREAD

**Issue by Brunzer** [PokemonGoMap/PokemonGo-Map#1159]
I'm using a mysql db to store data. I can see a few new tables in the database regarding gym stats (ie. gymmember, gympokemon). When looking at logs, I keep seeing the below exception thrown constantly.

```
2016-09-05 12:30:27,312 [    Thread-10588][           app][   ERROR] Exception on /raw_data [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1641, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1544, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1639, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1625, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/opt/local/PokemonGo-Map/pogom/app.py", line 98, in raw_data
    d['gyms'] = Gym.get_gyms(swLat, swLng, neLat, neLng)
  File "/opt/local/PokemonGo-Map/pogom/models.py", line 422, in get_gyms
    for d in details:
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 2967, in __iter__
    return iter(self.execute())
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 2960, in execute
    self._qr = ResultWrapper(model_class, self._execute(), query_meta)
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 2656, in _execute
    return self.database.execute_sql(sql, params, self.require_commit)
  File "/usr/local/lib/python2.7/dist-packages/playhouse/shortcuts.py", line 203, in execute_sql
    sql, params, require_commit)
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 3492, in execute_sql
    self.commit()
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 3316, in __exit__
    reraise(new_type, new_type(*exc_args), traceback)
  File "/usr/local/lib/python2.7/dist-packages/peewee.py", line 3485, in execute_sql
    cursor.execute(sql, params or ())
  File "/usr/local/lib/python2.7/dist-packages/pymysql/cursors.py", line 161, in execute
    result = self._query(query)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/cursors.py", line 317, in _query
    conn.query(q)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 837, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 1021, in _read_query_result
    result.read()
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 1304, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 983, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 395, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/err.py", line 120, in raise_mysql_exception
    _check_mysql_exception(errinfo)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/err.py", line 112, in _check_mysql_exception
    raise errorclass(errno, errorvalue)
ProgrammingError: (1146, u"Table 'pokemongomap.gymdetails' doesn't exist")
```

Everything still seems to work as expected unless you toggle Gyms to on in the Marker settings. With this toggled, pokemon do not spawn new or disappear when disappear time expires.Toggling Gyms back to off fixes this.

**Comment by Xcelled** [comment]
Duplicate of #1106

**Comment by Xcelled** [comment]
@justsomedudeonthenet Want to close this dup too?

**Comment by justsomedudeonthenet** [comment]
@Xcelled I sure do, thanks!

Closed as dup of #1106
